eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -17,15 +17,6 @@ steps:
     inputs:
       useGlobalJson: true
       performMultiLevelLookup: true
-  - task: UseDotNet@2
```

Comment:
Trying this because I couldn't get the dump to generate. From the docs, it seems that this feature requires .NET 5.0 SDK. We install both 6.0 and 3.1, but I'm not sure what the behavior is when both are installed side by side.

The blame-crash flag did at least generate a file that shows the offending test. Each time so far they have been different recorded tests from the Tables Extensions test project.

Reply:
Didn't help.

Reply:
I'm still wondering if this is because it is linux.

Reply:
It seems like it should be possible. There may be other undocumented args that need to be set - https://github.com/microsoft/vstest/issues/3085#issuecomment-961085082

Reply:
I think we may need to install dotnet-dump as well:
dotnet tool install --global dotnet-dump